nl.flyingpie.wtq.yml

```diff
@@ -0,0 +1,51 @@
+id: nl.flyingpie.wtq
+runtime: org.gnome.Platform
+runtime-version: "48"
+sdk: org.gnome.Sdk
+sdk-extensions:
+ - org.freedesktop.Sdk.Extension.dotnet9
+build-options:
+  prepend-path: "/usr/lib/sdk/dotnet9/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet9/lib"
+  prepend-pkg-config-path: "/usr/lib/sdk/dotnet9/lib/pkgconfig"
+
+command: wtq
+
+finish-args:
+  - --env=DOTNET_ROOT=/app/lib/dotnet           # .Net root
+  - --socket=wayland                            # So we can run the GUI
+  - --talk-name=org.freedesktop.Flatpak         # So we can start processes ("flatpak-spawn --host dolphin")
```

Comment:
So it only works with dolphin? What is this for anyways? Why do you need to launch dolphin?

Reply:
It can start applications, dolphin being an example. It's so you don't have to start an app first manually.

Reply:
yes if it is possible